omnibus/config/software/postgresql12.rb

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright:: Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "postgresql12"
```

Comment:
do we need this file anymore?

Reply:
I'll delete it and see what happens.

Reply:
Deleting... but is PG 12 supported?  If so, will deleting this create an error?  If not supported, let me know and I will remove 12 from 'supported versions' in another file.